Assets/Mirror/Runtime/NetworkBehaviour.cs

```diff
@@ -5,6 +5,11 @@
 
 namespace Mirror
 {
+    /// <summary>
+    /// Sync to everyone, or only to owner.
+    /// </summary>
+    public enum SyncMode { Everyone, Owner }
```

Comment:
I think "Observers"  will be better than "Everyone"

Reply:
true